modules/cardano-node.nix

```diff
@@ -187,6 +187,7 @@ in {
         key = keyId + ".sk";
       in ''
         [ -f /run/keys/${keyId} ] && cp /run/keys/${keyId} ${stateDir}${key}
+        ulimit -c unlimited
```

Comment:
Why is this needed? Would be good to have a comment. Is it save to have it on for production?